src/web/core/src/main/java/org/geoserver/web/data/resource/ResourceConfigurationPage.java

```diff
@@ -68,8 +68,7 @@ public ResourceConfigurationPage(String workspaceName, String layerName) {
                 // with a layer page open in another tab/window
                 throw new RuntimeException("Could not find workspace " + workspaceName);
             }
-            String nsURI = ns.getURI();
-            layer = getCatalog().getLayerByName(new NameImpl(nsURI, layerName));
+            layer = getCatalog().getLayerByName(ns.getName() + ":" + layerName);

```

Comment:
This change is interesting.. so the other method would not work properly in the context of a isolated workspace? Ideally the two should behave the same, if not, some copius javadoc warning people should be added (and maybe some migration guide for code sitting outside GeoServer)

Reply:
I don't think there is a need for a migration guide, I made this change because I want to support both the isolated and non isolated workspaces behavior.

Javadoc of ``getLayerByName`` method updated.